backend/src/api/vulnerabilities.ts

```diff
@@ -126,20 +130,22 @@ class VulnerabilitySearch {
 }
 
 export const update = wrapHandler(async (event) => {
-  let where = {};
```

Comment:
Why did this previous logic not work?

Reply:
I was trying to do nested queries on the domain's organization, which didn't seem to work - do you know if there is any way to make that work?